modules/ROOT/pages/advanced/events.adoc

```diff
@@ -418,9 +430,10 @@ The following event is provided by the link:{baseurl}/plugins/opensource/charmap
 | Fired before a special character is inserted into the editor.
 |===
 
+[[commentsevents]]
 === Comments events
 
-The following event is provided by the link:{baseurl}/plugins/premium/comments/[Comments plugin].
+The following event is provided by the xref:plugins/premium/comments.adoc[Comments plugin].
```

Comment:
This xref doesn't resolve... I believe it should be this instead:
```suggestion
The following event is provided by the xref:plugins/premium/comments/index.adoc[Comments plugin].
```

Reply:
done